scripts/install_test_dependencies.sh

```diff
@@ -15,8 +15,19 @@
 
 set -ex
 
-sudo apt-get update
-sudo apt-get install -y php5-cli
+if [ "${CIRCLECI}" != "true" ]; then
+    sudo apt-get update
+    sudo apt-get install -y php5-cli
```

Comment:
This script is only for CI systems. You don't have to run this script locally. Locally you can install them in whatever way you like.

Are you comfortable if it runs only on Jenkins?